README.md

```diff
@@ -98,4 +102,10 @@ maintained by the hypervisor.
 
 Since a hypervisor doing such code integrity checks will refuse to accept code pages that aren't signed by
 a key that the hypervisor trusts, this does impact eBPF programs running natively.  As such, when HVCI
-is enabled, eBPF programs work fine in interpreted mode, but not when using JIT compilation.
+is enabled, eBPF programs work fine in interpreted mode, but not when using JIT compilation because the JIT
+compiler does not have a key that the hypervisor trusts.  And since interpreted
+mode is absent in release builds, neither mode will work on an HVCI-enabled production system.
+
+Instead, a third mode is also supported by eBPF for Windows, in addition to JIT compiled and interpreted modes.
+This third mode entails compiling eBPF programs into regular Windows drivers that can be accepted by HVCI.
+For more discussion, see the [Native Code Generation documentation](docs/NativeCodeGeneration.md).
```

Comment:
This link gives page not found error.

Reply:
looks like this is not showing as this PR is not yet merged to main. Please ignore.